resolvers/user.ts

```diff
@@ -126,6 +128,27 @@ export class UserResolver {
         return noteResolvers;
     };
 
+    /**
+     * get notes that are created by this user from Backend REST API
+     */
+    getNotesFromRestAPI = async (args: { limit: number; page?: number }) => {
```

Comment:
@herparis Instead of changing `getNotes`, I created a new method so that callers of `getNotes` in `web-app` does not need to refactor.